Extensions/ParticleSystem/JsExtension.cpp

```diff
@@ -28,7 +28,7 @@ class ParticleSystemJsExtension : public gd::PlatformExtension {
         .AddIncludeFile(
             "Extensions/ParticleSystem/particleemitterobject-pixi-renderer.js")
         .AddIncludeFile(
-            "Extensions/ParticleSystem/pixi-particles-pixi-renderer.min.js")
+            "Extensions/ParticleSystem/pixi-particles.min.js")
```

Comment:
It's important to name files with pixi-renderer otherwise they are included in the Cocos2d-JS build.